Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+/*
+ * Requires: https://github.com/RedHatInsights/insights-pipeline-lib
+ */
+
+@Library("github.com/RedHatInsights/insights-pipeline-lib") _
+
+node {
+    cancelPriorBuilds()
+
+    runIfMasterOrPullReq {
+        runStages()
+    }
+}
+
+
+def runStages() {
+    openShift.withNode(image: "docker-registry.default.svc:5000/jenkins/jenkins-slave-base-centos7-ruby25-openscap:latest") {
+
+        scmVars = checkout scm
+        
+        stage('Install gems') {
+            sh "bundle install --path ./bundle"
+        }
+
+        // stage('Lint') {
+        //     sh "bundle exec rake validate"
+        // }
+
+        stage('Prepare the db') {
+            sh "bundle exec rake db:migrate --trace"
+            sh "bundle exec rake db:test:prepare"
+        }
+
+        stage('UnitTest') {
+            withStatusContext.unitTest {
+                sh "bundle exec rake test"
```

Comment:
Could you make this 'rake validate'? That runs rubocop in addition to just tests